full-backend-tests/src/test/java/org/graylog/plugins/views/aggregations/SearchWithAggregationsSupportingMissingBucketsIT.java

```diff
@@ -134,6 +135,39 @@ void testSingleFieldAggregationHasProperMissingBucket() {
 
     }
 
+    @ContainerMatrixTest
+    void testSingleFieldAggregationHasNoMissingBucketWhenSkipEmptyValuesIsUsed() {
+        final Pivot pivot = Pivot.builder()
+                .rollup(true)
+                .series(Count.builder().build(), Average.builder().field("age").build())
+                .rowGroups(Values.builder().field("firstName").limit(8).skipEmptyValues().build())
+                .build();
+        final ValidatableResponse validatableResponse = execute(pivot);
+
+        //General verification
+        validatableResponse.rootPath(PIVOT_RESULTS_PATH)
+                .body(".rows", hasSize(4))
+                .body(".total", equalTo(5))
+                .body(".rows.find{ it.key[0] == 'Joe' }", notNullValue())
+                .body(".rows.find{ it.key[0] == 'Jane' }", notNullValue())
+                .body(".rows.find{ it.key[0] == 'Bob' }", notNullValue())
+                .body(".rows.find{ it.key[0] == '" + MISSING_BUCKET_NAME + "' }", nullValue())
+                .body(".rows.find{ it.key == [] }", notNullValue());
+
+        //Empty bucket verification (should precede the last/total one - index 3)
+        //The only message with "empty" first name in a fixture is {(...)"lastName": "Cooper","age": 60(...)}
+        validatableResponse.body(".rows[3].key", not(contains("(Empty Value)")));
```

Comment:
Doesn't it repeat the check from line 154?

Reply:
Kinda, but with a difference: Line 154 checks if the first element of the key is `MISSING_BUCKET_NAME`, while this line is checking if a specific row contains it in any of its keys. (We should reuse the constant here though, fixing this)